Woof/WoofTests/Helper/AppAlert/AppAlertDescriptionTests.swift

```diff
@@ -0,0 +1,13 @@
+import XCTest
+
+final class AppAlertDescriptionTests: XCTestCase {
+    func testAppAlertDescriptionReturnsExpectedString() {
+        // Given
+        let logOut = "Do you really want to log out?"
+        let error = "Error"
+
+        // When // Then
+        XCTAssertEqual(AppAlert.logOut.rawValue, logOut)
```

Comment:
Could we simplify it? We do not have to create constants and call them only once as we can compare values in asserts. WDY?
```suggestion
        let logOut = "Do you really want to log out?"
        let error = "Error"

        // When // Then
        XCTAssertEqual(AppAlert.logOut.rawValue, "Do you really want to log out?")
```

Reply:
Fixed with 03725e9